build.sc

```diff
@@ -38,7 +38,7 @@ trait CommonModule extends ScalaModule {
 }
 trait CommonPublishModule extends CommonModule with PublishModule with CrossScalaModule{
   def publishVersion = "1.2.2"
-  def isDotty = crossScalaVersion.startsWith("0")
+  def isDotty = crossScalaVersion.startsWith("0") || crossScalaVersion.startsWith("3")
```

Comment:
nitpick: should we rename this to `isScala3`?

Reply:
Yep, this should be done at some point.